workflow/initiator_group_test.go

```diff
@@ -0,0 +1,144 @@
+package workflow
+
+import (
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/client/v1/nimbleos"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/param"
+	"github.com/hpe-storage/nimble-golang-sdk/pkg/service"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/suite"
+	"testing"
+)
+
+var initiatorGroupName = "InitiatorGroupTest"
```

Comment:
Should it be const instead of var?  This comment also applies in a few other places.